Orchestrator/v0.2/nlr_versions.json

```diff
@@ -24,10 +24,16 @@
       "description": "Bot Framework SDK release 4.10 - English ONNX V1.4 12-layer per-token intent base model",
       "minSDKVersion": "4.10.0"
     },
-    "pretrained.20210218.microsoft.dte.00.12.bert_example_ner.en.onnx": {
-      "releaseDate": "02/18/2021",
-      "modelUri": "https://aka.ms/pretrained.20210218.microsoft.dte.00.12.bert_example_ner.en.onnx",
-      "description": "(experimental) Bot Framework SDK release 4.12 - English ONNX V1.4 12-layer per-token entity base model",
+	"pretrained.20210521.microsoft.dte.01.06.int.en.onnx": {
+      "releaseDate": "06/14/2021",
+      "modelUri": "https://models.botframework.com/models/dte/onnx/pretrained.20210521.microsoft.dte.01.06.int.en.onnx.zip",
+      "description": "Bot Framework SDK release 4.10 - English ONNX V1.4 6-layer int per-token intent base model with CDF threshold config",
```

Comment:
Shall we elaborate what this threshold is for?

Reply:
CDF is kind of confusing for customers. I think I will just change it for "with updated calibration thresholds."